config/model_configs/sphere_baroclinic_wave_rhoe_hughes2023.yml

```diff
@@ -0,0 +1,16 @@
+dt_save_state_to_disk: "2days"
+initial_condition: "DryBaroclinicWave"
```

Comment:
Should this be MoistBaroclinicWave? Also are the initial condition exactly the same, or is there any perturbation related to topography?